### ISSUE TITLE
Plugin is not using locally installed node / npm installations

### ISSUE THREAD

**Issue by mativait** [eirslett/frontend-maven-plugin#124]
During debugging another issue ( node/npm memory exhaustion ) we discovered that frontend-maven-plugin is not using locally installed node/npm versions. 
How to reproduce :
1) clean workspace , remove node & node_packages & target folders
2) remove system-wide  node & npm  ( sudo apt-get remove node npm # in debian/ubuntu ) if installed
3) run maven package
4) experience error 
5) clean workspace again ( 1 )
6) install system-wide node & npm (sudo apt-get install npm node # in debian/ubuntu) if installed
7) run maven package
8) build succeeds.

**Comment by dg1202** [comment]
I've recently run into this same issue. I am using Vagrant with my repo in the shared directory. This was so I could have maven and the rest of my dev environment isolated, but hack on the project using my local IDE. 

What I think is happening is that once I kick of `mvn clean install

**Comment by dg1202** [comment]
So weird. Woke up this morning, and now everything works as expected??? (funny how that works sometimes)

 Ill keep digging and see where I was going wrong or what might have been. I simply blew away the node and node_modules it pull down during the build and tried again. Now it works. 

I did remove the global install of node but I swear I got the same result. Ill come back and post any findings I might discover. 

Using Vagrant and have my repo on the shared directory. I can issue `mvn clean install` on the VM CLI and get a full build, or specify `gulp build` within my Webstorm IDE and just build my front-end application, no scripts required. Working like a charm today!
